CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 ## 0.18.0 (Unreleased)
+
```

Comment:
@phamann do we need this `0.18.0` at the top still?

Reply:
Yep.

Reply:
Nope, looks like we don't: https://github.com/terraform-providers/terraform-provider-fastly/commit/820b30dd51e94bcb106a0b48938b96ac430ecbe8